source/_integrations/qbus.markdown

```diff
@@ -0,0 +1,76 @@
+---
+title: Qbus
+description: Instructions on how to integrate your Qbus installation with Home Assistant.
+ha_category:
+  - Switch
+ha_platforms:
+  - switch
+ha_iot_class: Local Push
+ha_codeowners:
+  - '@Qbus-iot'
+  - '@thomasddn'
+ha_release: 2024.12
+ha_domain: qbus
+ha_integration_type: device
+ha_config_flow: true
+---
+
+The **Qbus** {% term integration %} allows you to integrate your [Qbus Control](https://www.qbus.be) into Home Assistant. **Qbus** is a Belgian manufacturer of Home Automation systems.
+
+## Prerequisites
+
+This integration communicates with a **Qbus** controller over an MQTT server.
+
+The controllers cannot communicate directly with MQTT. Therefore, we developed a gateway which you must install before enabling this Integration. Our gateway is a software tool that runs on all Linux platforms. It can simply be installed by running a script or a Docker container. For detailed instructions, please refer to the [Qbus MQTT Gateway documentation](https://github.com/Qbus-iot/qbus-mqttgw).
```

Comment:
```suggestion
The controllers cannot communicate directly with MQTT. Therefore, you need to install the Qbus gateway before enabling this integration. The Qbus gateway is a software tool that runs on all Linux platforms. It can be installed by running a script or a Docker container. For detailed instructions, please refer to the [Qbus MQTT Gateway documentation](https://github.com/Qbus-iot/qbus-mqttgw).
```